torchvision/transforms/transforms.py

```diff
@@ -331,31 +338,39 @@ def __call__(self, img):
         Returns:
             PIL Image: Randomly flipped image.
         """
-        if random.random() < 0.5:
+        if random.random() < self.p:
             return F.hflip(img)
         return img
 
     def __repr__(self):
-        return self.__class__.__name__ + '()'
+        return self.__class__.__name__ + '(p={})'.format(self.p)
 
 
 class RandomVerticalFlip(object):
-    """Vertically flip the given PIL Image randomly with a probability of 0.5."""
+    """Vertically flip the given PIL Image randomly with a probability.
+
+    Args:
+        p (float): probability of image is flipped. Default value is 0.5
+    """
+
+    def __init__(self, p=0.5):
+        self.p = p
 
     def __call__(self, img):
         """
         Args:
             img (PIL Image): Image to be flipped.
+            proba (float): probability of image is flipped. Default value is 0.5
```

Comment:
the probability is not an argument to the `__call__` method, right?

Reply:
Yes, sorry, forgot to remove this